tool/gravity/cli/config.go

```diff
@@ -355,52 +401,14 @@ func (i *InstallConfig) NewInstallerConfig(
 	env *localenv.LocalEnvironment,
 	wizard *localenv.RemoteEnvironment,
 	process process.GravityProcess,
```

Comment:
We still to run some tasks in the server as it's when it has access to the wizard operator.